blueprints/attach-disk.yaml

```
@@ -1,36 +1,12 @@
 tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
-  - http://cloudify.co/spec/cloudify/4.5/types.yaml
-  - plugin.yaml
-  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-utilities-plugin/1.16.1/plugin.yaml
+  - http://www.getcloudify.org/spec/cloudify/6.1.0/types.yaml
```

זה בסדר אבל אידיאלי היינו עושים ככה:
```suggestion
  -   - https://cloudify.co/spec/cloudify/6.1.0/types.yaml

```